src/tests/refinement/Helper.rs

```diff
@@ -22,29 +22,33 @@ pub fn json_refinement_check(PATH: &str, QUERY: &str) -> bool {
 
 pub fn xml_run_query(PATH: &str, QUERY: &str) -> QueryResult {
```

Comment:
I like that we are using unwrap here since it is only called from `cargo test`, good idea!